src/stories/Home.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Canvas } from '@storybook/addon-docs/blocks';
+
+<Meta title="Home" />
+
+# H5Web Component Library
+
+A library of React components and utilities for data visualization, based on [react-three-fiber](https://github.com/pmndrs/react-three-fiber).
+
+The components are organised in two categories:
+
+- 🎨 **Visualizations**: the top-level visualization components — i.e. the components you will most likely need.
+- 🧱 **Building Blocks**: the low-level components used by the visualization components — for advanced uses only.
+
+## Getting started
+
+1. Make you sure you have `react` and `react-dom` v16 or greater installed.
+1. Install the library:
+
+   ```bash
+   npm install @h5web/lib
+   ```
+
+1. Import and use a visualization component:
+
+   ```tsx
+   import { ScalarVis } from '@h5web/lib';
+
+   // ...
+
+   <ScalarVis value={1024} />;
+   ```
+
+1. Other visualization components expect to receive an `ndarray` and a domain as props.
+
+   1. Install `ndarray` and its corresponding types package (if you use TypeScript):
+
+      ```bash
+      npm install ndarray
+      npm install --save-dev @types/ndarray
+      ```
+
+   1. Pass an `ndarray` and domain to the visualization:
+
+      ```tsx
+      import ndarray from 'ndarray';
+      import { HeatmapVis, findDomain } from '@h5web/lib';
+
+      // Source nD array
+      const values = [
+        [0, 1, 2],
+        [3, 4, 5],
+      ];
+
+      // Flatten array
+      const flatArray: number[] = values.flat(Infinity);
+
+      // Convert to ndarray and find domain
+      const dataArray = ndarray<number>(values, [2, 3]);
+      const domain = findDomain(values);
+
+      // ...
+
+      <HeatmapVis dataArray={dataArray} domain={domain} />;
+      ```
+
+1. Configure the visualization:
+
+   ```tsx
+   <HeatmapVis
+     dataArray={dataArray}
+     domain={domain}
+     colorMap="Inferno"
+     showLoader={false}
```

Comment:
We might as well put every possible argument: `scaleType`, `showGrid`, `keepAspectRatio`

Reply:
Hmm, the idea here is just to give an example, not to document the Heatmap, which is done by the Heatmap's stories ("Doc" tab).

Reply:
Yes I get it but I think it is better to give a complete example from the get-go. It doesn't add much lines and will save someone a few seconds :slightly_smiling_face:

Reply:
I don't know, I don't see the relevance of passing the props you mention, since they have sensible defaults -- for instance, passing `keepAspectRatio` would be redundant, and `keepAspectRatio={false}` would not make for a good first example. Moreover, if users have TypeScript, they'll get autocompletion on props, anyway.

Reply:
Maybe just `scaleType` to show how to use the `ScaleType` enum?

Reply:
I disagree: I always find frustrating to have an example with partial arguments as I have to go check in the full doc/signature. My point of view is either, give me only the required arguments or give me all of them. 

Sure, autocompletion helps but I can imagine some situations where it will not work.

Your call but at least adding `scaleType` would be a nice touch :+1:

Reply:
I'll add `scaleType`, then 👍